docs/index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html class="scroll-smooth" lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+  </head>
+  <body>
+    <header class="text-gray-400 bg-gray-800 body-font sticky top-0 font-sans">
+      <div
+        class="container mx-auto flex flex-wrap p-5 flex-col md:flex-row items-center"
+      >
+        <nav
+          class="flex lg:w-2/5 flex-wrap items-center text-base md:ml-auto text-transparent bg-clip-text bg-gradient-to-br from-green-400 to-green-600"
+        >
+          <a class="mr-5 hover:text-green-500" href="index.html">Home</a>
+
+          <a class="mr-5 hover:text-green-500" href="#features">Features</a>
+          <a class="mr-5 hover:text-green-500" href="#contributing"
+            >Contribute</a
+          >
+
+          <a class="hover:text-green-500" href="#support">Support</a>
+        </nav>
+        <a
+          class="flex order-first lg:order-none lg:w-1/5 title-font font-medium items-center text-white lg:items-center lg:justify-center mb-4 md:mb-0"
+          href="index.html"
+        >
+          <img
+            class="rounded-full"
+            src="img/obsidian_logo.png"
+            alt="obsidian_logo"
+            width="50px"
+            height="50px"
+          />
+          <span
+            class="ml-3 text-xl xl:block lg:hidden font-bold text-transparent bg-clip-text bg-gradient-to-br from-green-400 to-green-600"
+            >Obsidian</span
+          >
+        </a>
+        <div class="lg:w-2/5 inline-flex lg:justify-end ml-5 lg:ml-0">
+          <button
+            class="bg-gradient-to-r from-green-400 to-green-600 border-0 py-2 px-8 focus:outline-none hover:from-cyan-500 hover:to-blue-500 text-white font-bold py-2 px-4 rounded-full"
+          >
+            Invite Now →
```

Comment:
You can add an arrow icon here instead of a unicode character

Reply:
Oh you mean an svg?

Reply:
yeah, you can use a svg or use free icons from fontawesome or some other website.